com.unity.render-pipelines.universal/Runtime/ShadowUtils.cs

```diff
@@ -102,9 +102,11 @@ public static bool ExtractPointLightMatrix(ref CullingResults cullResults, ref S
 
         public static void RenderShadowSlice(CommandBuffer cmd, ref ScriptableRenderContext context,
             ref ShadowSliceData shadowSliceData, ref ShadowDrawingSettings settings,
-            Matrix4x4 proj, Matrix4x4 view)
+            Matrix4x4 proj, Matrix4x4 view, float depthBias)
         {
-            cmd.SetGlobalDepthBias(1.0f, 2.5f); // these values match HDRP defaults (see https://github.com/Unity-Technologies/Graphics/blob/9544b8ed2f98c62803d285096c91b44e9d8cbc47/com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDShadowAtlas.cs#L197 )
+            cmd.SetGlobalDepthBias(depthBias, 2.5f); // these values match HDRP defaults (see https://github.com/Unity-Technologies/Graphics/blob/9544b8ed2f98c62803d285096c91b44e9d8cbc47/com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDShadowAtlas.cs#L197 )
```

Comment:
The settings don't match HDRP defaults? 
Also shouldn't we expose parameter for slopeBias as well?